### ISSUE TITLE
[0.14.1] Many problems of dependencies due to the rework

### ISSUE THREAD

**Issue by Maze-fr** [Myrmod/svelte-babylon#16]
Hello,

Many problems of broken dependencies when building with version 0.14.1
Simple way to reproduce, an empty project with just this :
```html
<script lang="ts">
    import { Canvas } from "svelte-babylon";
</script>
<Canvas>

</Canvas>
```
Then `npm run build`

**Comment by Myrmod** [comment]
already trying to figure this out, thanks for creating the issue

**Comment by Myrmod** [comment]
I just released version 0.14.2 with fixes for all the bugs that I could find.

I somehow cannot build the vitebook documentation anymore. So testing everything manually is a pain. I need to work on a proper and reliable documentation. Some kind of testing would be nice as well.
